helpertools/fastqc_module.py

```diff
@@ -1,22 +1,132 @@
-def gc_check(sequence, gc_bounds):  # accepts a string - sequence of nucleotides from FASQC-dictionary - and a tuple of too numeric values -  lower and upper bounds for gc-content in provided string
+def gc_check(sequence, gc_bounds):
+    '''
+    Checks if gc-content of the string is within set bounds
+    Parametrs
+    ---------------------
+    seqence : string
+        sequence of nucleotides from FASQC-dictionary
+    length_bounds : tuple
+        too numeric values -  lower and upper bounds for gc-content
+
+    Returns
+    ---------------------
+    bool
+
+    '''
+
     gc_count = 0
     gc_count = sum(True for nucl in list(sequence) if nucl == "G" or nucl == "C")
     gc = 0
```

Comment:
- выражение `nucl == 'G' or nucl == 'C'` уже возвращает True или False
- конвертировать в список не обязательно, можно сразу по строке проитерироваться
```suggestion
    gc = sum(nucl in "CG" for nucl in sequence) / len(sequence)
```